### ISSUE TITLE
"Basic plugin analysis failed" after upgrading to 0.3.11

### ISSUE THREAD

**Issue by avdv** [FriendsOfShopware/FroshPluginUploader#132]
In our CI/CD pipeline, we were using the uploader to publish plugin releases.

Now, we had 0.3.10 and suddenly the pipeline failed:
```
Fatal error: Uncaught TypeError: strtolower() expects parameter 1 to be string, null given in /app/src/Components/SBP/Components/Producer.php:42
```
Fair enough, I realised we need to update to 0.3.11. But lo and behold, this fails with:
```
$ php /app/bin/pluginupload ext:upload "${PLUGIN_NAME}-${PACKAGE_VERSION}".zip
 [ERROR] === basicCheck ===                                                     
         Basic plugin analysis failed.                                          
         new platform plugin system is only available for version >= 6          
```
What gives? Note that the plugin is indeed a platform plugin and publishing worked fine before.

Is there some reference where I can look what is checked and in which environment?

**Comment by shyim** [comment]
Thats now an issue from this uploader. This error comes from the API. You should have the same when you upload it manuelly

**Comment by avdv** [comment]
Nope, just tried it manually and it worked.

(logged in, chose the shopware 6 plugin, opened the relevant version and clicked on "Zip Archive aktualisieren", uploaded the zip artifact build by the CI pipeline:

![image](https://user-images.githubusercontent.com/3471749/115006303-02e29f00-9ea9-11eb-91b2-f515a16ca6f9.png)

**Comment by avdv** [comment]
In the static analysis, there is an error:
```
Warnings of the static code analysis:
The command "git checkout v6.4.0.0" failed.

Exit Code: 1(General error)
```
What does that mean?

Again, where is the spec? What is tested and how?

**Comment by shyim** [comment]
The static analyse is currently broken in the API. Is fixed with the 6.4.0.0 release.

The error: `new platform plugin system is only available for version` has been fixed on backend side of the API
